### PR TITLE
mslg CLI

### PR DESCRIPTION
**Pull request by Danieladu**
mslg cli tool

### CODE REVIEW

**Comment by vishwacsena** [comment]
@Danieladu is this ready for review? Not sure, so wanted to check in.

File: packages/luis/test/fixtures/luisgen/obj/Debug/netcoreapp2.1/LUISGenTest.AssemblyInfoInputs.cache [hunk 1]
```diff
@@ -0,0 +1 @@
+c9fa663dac6390a7eefc64231f367790aa287411
```
Comment (feich-ms) [inline packages/luis/test/fixtures/luisgen/obj/Debug/netcoreapp2.1/LUISGenTest.AssemblyInfoInputs.cache:1]: What is used for?

File: rush.json [hunk 2]
```diff
@@ -31,6 +31,10 @@
       "packageName": "@microsoft/bf-dispatcher",
       "projectFolder": "packages/dispatcher"
     },
+    {
+      "packageName": "@microsoft/bf-cli-lg",
+      "projectFolder": "packages/lg-cli"
```
Comment (feich-ms) [inline rush.json:36]: >lg-cli [](start = 33, length = 6)

change lg-cli to lg?

File: packages/lg-cli/test/commands/lg-cli/translate.test.ts [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+import {test} from '@oclif/test'
+import {TestUtil} from './test-util'
+import * as fs from 'fs-extra'
+import * as path from 'path'
+
+const  testcaseFolderPath = './../../fixtures/testcase'
+const  generatedFolderPath = './../../fixtures/generated'
+const  verifiedFolderPath = './../../fixtures/verified'
+const  generatedFolder = path.join(__dirname, generatedFolderPath)
+const translatekey = 'edabe9fb17244f6a92ec617867e6d7cd'
```
Comment (feich-ms) [inline packages/lg-cli/test/commands/lg-cli/translate.test.ts:17]: >edabe9fb17244f6a92ec617867e6d7cd [](start = 22, length = 32)

should not include the real key. Use something like nock to mock the translate result

Comment (boydc2014) [reply packages/lg-cli/test/commands/lg-cli/translate.test.ts:17]: no key should be checked in source code, and you should deprecate this key in Azure ASAP.

**Comment by vishwacsena** [comment]
@Danieladu this looks good to me. I pushed up some changes as I reviewed this. See my commit for delta. 

@munozemilio can you please review as well?

**Comment by vishwacsena** [comment]
@munozemilio how can we auto-generate command content for README? I put a placeholder with links to the .lg file format as well as added a doc to describe the translate functionality. README updates as well as CR from you are the only pending items for this PR.

**Comment by vishwacsena** [comment]
@Danieladu @feich-ms can one of you take a look at why tests are failing here? Would be good to get this sorted and merged in.

**Pull request merged**